manifests/c/CubeSoft/CubePDFUtility/1.6.3/CubeSoft.CubePDFUtility.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CubeSoft.CubePDFUtility
+PackageVersion: 1.6.3
+PackageLocale: en-US
+Publisher: CubeSoft, Inc.

```

```suggestion
Publisher: CubeSoft
```
This must match Control Panel.